dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/SecondaryVoltageControl/Simplified/SecondaryVoltageControl.mo

```diff
@@ -17,35 +17,35 @@ model SecondaryVoltageControl "Model for simplified secondary voltage control"
   import Dynawo.Types;
 
   //Regulation parameters
-  parameter Types.PerUnit Ck "Sensitivity of voltage to level variations in pu (base UNom)";
+  parameter Types.Time Alpha "PI integral gain";
```

Comment:
If this is a time constant, shouldn't you choose a name beginning by t ?

Reply:
It is not a time constant in fact, I change the type.